modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/version/DefaultVersionWithScopesTestCase.java

```diff
@@ -73,189 +64,157 @@
-            accessTokenGenerationResponse = new JSONObject(apiStore.generateUserAccessKey(consumerKey, consumerSecret,
-                                                                                          requestBody, tokenEndpointURL)
-                                                                   .getData());
-            accessToken = accessTokenGenerationResponse.getString("access_token");
+            requestBody = "grant_type=password&username=" + USER_MIKE + "@" + storeContext.getContextTenant().getDomain()
+                    + "&password=" + user_mike_password + "&scope=" + user_scope;
+            accessTokenGenerationResponse = new JSONObject(restAPIStore.generateUserAccessKey(consumerKey, consumerSecret,
+                    requestBody, tokenEndpointURL)
+                    .getData());
+            userAccessToken = accessTokenGenerationResponse.getString("access_token");
 
-            requestHeaders = new HashMap<String, String>();
-            requestHeaders.put("Authorization", "Bearer " + accessToken);
+            requestHeaders = new HashMap<>();
+            requestHeaders.put("Authorization", "Bearer " + userAccessToken);
 
             //Accessing GET method without the version in the URL using the token mike received.
-            response = HttpRequestUtil.doGet(gatewayUrlsWrk.getWebAppURLNhttp() + "defaultversionscope", requestHeaders);
+            response = HttpRequestUtil.doGet(apiInvocationUrl, requestHeaders);
             assertEquals(response.getResponseCode(), Response.Status.FORBIDDEN.getStatusCode(),
-                         "Mike should receive an HTTP 403 when trying to access"
-                         + " the GET resource. But the response code was " + response.getResponseCode());
+                    USER_MIKE + " should receive an HTTP 403 when trying to access"
+                            + " the GET resource. But the response code was " + response.getResponseCode());
         }
         //Catching generic Exception since apiPublisherApi and apiStore classes throw Exception from their methods.
         catch (Exception e) {
             log.error("Error while executing test case " + e.getMessage(), e);
-            //fail the test case.
-            assertTrue(false, e.getMessage());
+            fail(e.getMessage());
         }
     }
 
     @AfterClass(alwaysRun = true)
     public void destroy() throws Exception {
-
-        if (apiStore != null) {
-            apiStore.removeApplication(APP_NAME);
-        }
-
-        if (apiPublisher != null) {
-            apiPublisher.deleteAPI(API_NAME, API_VERSION, API_PROVIDER);
+        SubscriptionListDTO subsDTO = restAPIStore.getAllSubscriptionsOfApplication(applicationID);
+        for (SubscriptionDTO subscriptionDTO : subsDTO.getList()) {
+            restAPIStore.removeSubscription(subscriptionDTO.getSubscriptionId());
         }
+        restAPIStore.deleteApplication(applicationID, null);
```

Comment:
This method is updated by removing the tenant domain. Lets remove the null parameter.

Reply:
Done.